packages/logs/src/boot/startLogs.ts

```diff
@@ -87,6 +90,11 @@ export function doStartLogs(
     )
   }
   errorObservable.subscribe(reportError)
+  if (isExperimentalFeatureEnabled('forward-logs')) {
+    trackConsoleLogs(['log', 'info', 'debug', 'warn'], (log: ConsoleLog) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏on a design perspective, it seems odd that `trackConsoleLogs` call is not colocated with `trackConsoleError`, `trackRuntimeError`, ... calls, wdyt?

Reply:
I agree. I moved `trackConsoleLogs` next to them.
However, someone could argue that [logger.error](https://github.com/DataDog/browser-sdk/blob/b1128af07c5b4498ad036a38ab3417e3bc446065/packages/logs/src/boot/startLogs.ts#L74) and [logger[log.status] ](https://github.com/DataDog/browser-sdk/blob/b1128af07c5b4498ad036a38ab3417e3bc446065/packages/logs/src/boot/startLogs.ts#L37) aren't collocated anymore. To me it's good enough, but let me know.